slick.grid.js

```diff
@@ -3749,12 +3782,80 @@ if (typeof Slick === "undefined") {
       selectionModel.setSelectedRanges(rowsToRanges(rows));
     }
 
-    function isGroupNode (row, cell) {
+    function isGroupNode(row, cell) {
       return $(getCellNode(row, cell))
         .parents('.slick-group')
         .length > 0;
     }
 
+    function getHiddenCssClass(index) {
+      var column = columns[index];
+      if (!column.hidden) return null;
+      if (column.showHidden) return 'show-hidden';
+      return 'hidden';
+    }
+
+    function getColumnVisibleWidth(column) {
+      return isColumnVisible(column) ? column.width : 0;
+    }
+
+    function refreshColumns() {
+      setColumns(columns);
+    }
+
+    function hideColumn(column) {
+      column.hidden = true;
+      delete(column.showHidden);
+    }
+
+    function unhideColumn(column) {
+      delete(column.hidden);
+      delete(column.showHidden);
+    }
+
+    function showHiddenColumnsByDirection(column, columnDirection) {

```

Comment:
it's not immediately obvious what `columnDirection` is. Can you drop a comment to help the consumer here?